### ISSUE TITLE
Spacesuits and Node.js - How open source will change NASA

### ISSUE THREAD

**Issue by danmactough** [NodeDC/talks#6]
**Your name:** Collin Estes

**Your idea:** Been having convos with @dshaw (Dan) and @mklumpkin (Kyle) from NodeSource, and Kyle suggested this talk to me via email. He said @CollinEstes is interested in presenting it again and that NodeSource could bring him here to present at a meetup (or something like along those lines). It was previously presented EnterpriseJS Austin -- video here https://vimeo.com/168064722 

**Approx length:** ~20  minutes
## 

2015 saw the creation of a new enterprise architecture at NASA’s Johnson Space Center, built on open source technologies like Elasticsearch, RethinkDB, Node.js, and Docker. This year our Node.js services are integrating and visualizing data which helps keep our astronauts safe, while providing the backbone of the software services that will drive the development of new spacesuit technologies to take us to Mars and beyond.

In this talk, Collin discussed how this one data integration solution will help kickstart an open source revolution, and changed how software is created at NASA. A solution which has earned NASA’s sole nomination for the ACT-IAC Igniting Innovation 2016 award.

Collin Estes is the Director of Software Engineering for MRI Technologies in Houston TX. MRI Technologies is a woman-owned small business serving NASA since 1988. Collin has been leading the software engineering for the EMU spacesuit contract at Johnson Space Center since 2006. In addition to working for NASA, Collin leads the MRI Dev Team which works with small and medium size businesses to build custom business applications primarily written in Node.js. Collin leads both self-directed teams primarily as a full stack engineer while also serving as the lead. Within NASA Collin works to promote internal open source and collaboration across the agency, bringing together development teams from all the various NASA centers and divisions.

**Comment by CollinEstes** [comment]
Hello all, I actually will be at NASA HQ in DC from 8/29 - 9/1.  Not sure what night of the week you typically meet but that might be an opportunity for me to come to meetup one of those nights.

**Comment by danmactough** [comment]
@CollinEstes Hmmm. Is Thursday, 9/1 a possibility?

**Comment by CollinEstes** [comment]
@danmactough yeah actually that might work, I should know for sure here on Monday.  I can circle back once I know.

**Comment by CollinEstes** [comment]
Hello everyone, I'm sorry for the late notice but I actually will be in DC the week of the 29th through the 1st.  

Is there a planned meetup that Thursday night?  I'd be interested in coming if so.

**Comment by danmactough** [comment]
Hey @CollinEstes, we don't have a regular meetup that Thursday but if you want to give your talk, we can organize one.
